app/assets/stylesheets/partials/_cafe.scss

```diff
@@ -0,0 +1,30 @@
+.cafe {
+  &.working {
+    color: $fsek-orange;
+    display: block;
+    font-weight: bold;
+    margin: 20px 0 20px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `20px 0` instead of `20px 0 20px 0`